docs/source/examples/simulate_RFB_example.ipynb

```diff
@@ -56,7 +56,7 @@
    "metadata": {},
    "source": [
     "Below, we define a cell with a 5 mL CLS, 10 mL NCLS, and 10 mM initial concentrations for all four species ($C_{ox,\\scriptscriptstyle CLS},~C_{red,\\scriptscriptstyle CLS},~C_{ox,\\scriptscriptstyle NCLS},~C_{red,\\scriptscriptstyle NCLS}$). The voltage difference between the formal reduction potential ($E^{o^{\\prime}}$) of the posolyte and negolyte species is 1.0 V, defined as $E_{cell}^{o^{\\prime}}=E_{\\scriptscriptstyle posolyte}^{o^{\\prime}}-E_{\\scriptscriptstyle negolyte}^{o^{\\prime}}$. \n",
-    "A cell ohmic resistance of 1.0 $\\Omega$, and electrochemical rate constants ($k_{o}$) of $10^{-3}$ cm/s for both posolyte and negolyte species, are also defined. The `ZeroDModel` class defaults to one-electron transfers for redox-active species, but this can be changed as shown in **link to redox flow cell docs**. Additionally, the simulation timestep `time_increment` defaults to 0.01 seconds."
+    "A cell ohmic resistance of 1.0 $\\Omega$, and electrochemical rate constants ($k_{o}$) of $10^{-3}$ cm/s for both posolyte and negolyte species, are also defined. The [`ZeroDModel`](https://ericfell-rfbzero.readthedocs.io/en/latest/flowcell.html#src.rfbzero.redox_flow_cell.ZeroDModel) class defaults to one-electron transfers for redox-active species, but this can be changed as shown in the [documentation](https://ericfell-rfbzero.readthedocs.io/en/latest/flowcell.html). Additionally, the simulation timestep [`time_increment`](https://ericfell-rfbzero.readthedocs.io/en/latest/flowcell.html#src.rfbzero.redox_flow_cell.ZeroDModel) defaults to 0.01 seconds."
```

Comment:
We probably don't need a link for `time_increment`

Reply:
done